A lire : \
https://pypi.org/project/superalgo/ \
https://github.com/tesseract-ocr/tesseract \
https://pypi.org/project/pyspellchecker/ \
https://pypi.org/project/spell-checker/ 

Si image déjà téléchargée

In [3]:
pip install pytesseract Pillow

Note: you may need to restart the kernel to use updated packages.


In [1]:
import pytesseract
from PIL import Image

# Open the image file
image = Image.open('photo_essai_tesseract.jpg')

# Use pytesseract to do OCR on the image
text = pytesseract.image_to_string(image)

# Print the extracted text
print(text)

  

—_—

>uM

= em Om

 

2 ae teh RN SRE A ER Mr: AN RS

| =
FR ety nate em ae ES
Ss

mE mecaeNRR I NE EF

[xvscper 33.]

|
'
‘

Monday, December 10, 1798.

Sn a ie selene aera ee

FOR THE GAZETTE,

 

MR. JENKs,

‘Tuar there fhould exift in this free
and enlightened country the fentiments imputed
to our Jacobins, fo extremely hoftile to all re-
ligion, and to every regular’ conftituted Govern.
ment, and fo daringly bent on the propagation

a

ener

wonder, and puzzled the underftandings of our
frofound.it politicians. The publication of
Robifon's Proofs of a Confpiracy, in my opinion,
removes every doubt. ‘This author, deferving
of the patronage of every American, has traced
this pation for insovation to its true fource,
and folemnly warned us of what unthinking

mortals very little expect. I will endeavor, for
the benefit of your readers, to trace in thefe

proofs the prefent diforders of the world.

_ Thenations of Europe have been infelted for
about a century pait with a fet of ph

In [8]:
import pytesseract
from PIL import Image

def perform_ocr(image_path):
    # Open the image file
    image = Image.open(image_path)

    # Use pytesseract to do OCR on the image and get the OCR result with confidence scores
    result = pytesseract.image_to_data(image, output_type=pytesseract.Output.DICT)

    # Extract confidence scores and filter out empty strings
    confidences = [int(conf) for conf in result['conf'] if conf]

    # Calculate the average confidence score
    if confidences:
        average_confidence = sum(confidences) / len(confidences)
    else:
        average_confidence = 0

    # Normalize the average confidence score to a percentage scale
    certainty_percentage = (average_confidence / 100) * 100

    # Get the recognized text
    recognized_text = pytesseract.image_to_string(image)

    return recognized_text, certainty_percentage

# Example usage
image_path = 'photo_essai_tesseract.jpg'
recognized_text, certainty_percentage = perform_ocr(image_path)

print("Recognized text:", recognized_text)
print("Certainty percentage:", certainty_percentage)

Recognized text:   

—_—

>uM

= em Om

 

2 ae teh RN SRE A ER Mr: AN RS

| =
FR ety nate em ae ES
Ss

mE mecaeNRR I NE EF

[xvscper 33.]

|
'
‘

Monday, December 10, 1798.

Sn a ie selene aera ee

FOR THE GAZETTE,

 

MR. JENKs,

‘Tuar there fhould exift in this free
and enlightened country the fentiments imputed
to our Jacobins, fo extremely hoftile to all re-
ligion, and to every regular’ conftituted Govern.
ment, and fo daringly bent on the propagation

a

ener

wonder, and puzzled the underftandings of our
frofound.it politicians. The publication of
Robifon's Proofs of a Confpiracy, in my opinion,
removes every doubt. ‘This author, deferving
of the patronage of every American, has traced
this pation for insovation to its true fource,
and folemnly warned us of what unthinking

mortals very little expect. I will endeavor, for
the benefit of your readers, to trace in thefe

proofs the prefent diforders of the world.

_ Thenations of Europe have been infelted for
about a century pait

Certainty percentage: 70.26853084775854 TESSERACT

In [1]:
pwd

'/home/asalva/memoire_hn1'

Spellchecker de Python

In [10]:
!pip install spellchecker #spell-checker or spellchecker?

     |████████████████████████████████| 3.9 MB 6.4 MB/s eta 0:00:01     |█████████████████               | 2.1 MB 6.4 MB/s eta 0:00:01
  Created wheel for spellchecker: filename=spellchecker-0.4-py3-none-any.whl size=3966515 sha256=1b0c3ae470202542b5c730c9404f539c37b3abc988f71afd59dbcee0f72b371f
  Stored in directory: /home/asalva/.cache/pip/wheels/f5/e3/fd/9d49c61c34dd99995c2d38f6b94fd33bfa7a559e3bfd8f3041
  Created wheel for inexactsearch: filename=inexactsearch-1.0.2-py3-none-any.whl size=7121 sha256=1362a3da52ef4cebff029461220defa3df2576588669baa4abaf4b5a6dd47c4c
  Stored in directory: /home/asalva/.cache/pip/wheels/11/c0/e2/2475e4b2981445a2a1f469c7a88ff3e4f1ce7acc6281e56033
  Created wheel for silpa-common: filename=silpa_common-0.3-py3-none-any.whl size=8468 sha256=24887293bdff52a0da17765284834cf5c4ce21200b1d6a147f17ca4fc93b51a0
  Stored in directory: /home/asalva/.cache/pip/wheels/c7/a5/67/816bbc59aef9be86b1d58f1db723f8b5896ee9605ea0603b5b
  Created wheel for soundex: filename=s

In [2]:
from spellchecker import SpellChecker

def spell_check_text(input_file, output_file, language='en'):
    # Create an instance of SpellChecker
    spell = SpellChecker(language=language)

    # Read input file
    with open(input_file, 'r', encoding='utf-8') as file:
        text = file.read()

    # Split the text into words
    words = text.split()

    # Find misspelled words
    misspelled = spell.unknown(words)

    # Correct misspelled words
    for word in misspelled:
        corrected_word = spell.correction(word)
        if corrected_word is not None:
            text = text.replace(word, corrected_word)

    # Write corrected text to output file
    with open(output_file, 'w', encoding='utf-8') as file:
        file.write(text)

    print("Spell-checking completed. Corrected text saved to:", output_file)

# Example usage
input_file = 'essai1.txt'
output_file = 'essai1_corrected_output.txt'
spell_check_text(input_file, output_file)

Spell-checking completed. Corrected text saved to: essai1_corrected_output.txt


In [11]:
from speller3x.checker import Checker

# Define the path to the text file
file_path = 'essai1.txt'

# Read the content of the text file
with open(file_path, 'r', encoding='utf-8') as file:
    text = file.read()

# Create a Checker instance and perform spell-checking
checker = Checker(text, "english")
errors = checker.check()

# Print the spell-checking results
print("Spell-checking results:")
for error in errors:
    print("Word:", error.word)
    print("Suggestions:", error.suggestions)
    print("Context:", error.context)
    print()

ModuleNotFoundError: No module named 'speller3x'

In [2]:
!pip install --upgrade spellchecker

Requirement already up-to-date: spellchecker in /home/asalva/.local/lib/python3.8/site-packages (0.4)


In [3]:
from spellchecker import SpellChecker

# Define the path to the text file
file_path = 'essai1.txt'

# Read the content of the text file
with open(file_path, 'r', encoding='utf-8') as file:
    text = file.read()

# Create an instance of SpellChecker
spell = SpellChecker()

# Split the text into words
words = text.split()

# Find misspelled words
misspelled = spell.unknown(words)

# Calculate the total number of words
total_words = len(words)

# Calculate the number of misspelled words
num_misspelled = len(misspelled)

# Calculate the number of correctly spelled words
num_correct = total_words - num_misspelled

# Calculate the percentage of correct words
percentage_correct = (num_correct / total_words) * 100

# Calculate the proportion of errors
proportion_errors = (num_misspelled / total_words)

print("Total words:", total_words)
print("Number of misspelled words:", num_misspelled)
print("Number of correctly spelled words:", num_correct)
print("Percentage of correct words:", percentage_correct)
print("Proportion of errors:", proportion_errors)

ModuleNotFoundError: No module named 'indexer'

Essai avec spell-checker

In [8]:
!pip install python-algorithms-3x
!pip install spell-checker

ERROR: Could not find a version that satisfies the requirement python-algorithms-3x (from versions: none)
ERROR: No matching distribution found for python-algorithms-3x
  Using cached spell_checker-1.6.1-py3-none-any.whl (2.7 MB)
  Using cached data_structures3x-1.4.6-py3-none-any.whl (6.3 kB)
ERROR: Could not find a version that satisfies the requirement python-algorithms-3x (from spell-checker) (from versions: none)
ERROR: No matching distribution found for python-algorithms-3x (from spell-checker)


Pour résoudre problème d'incompatibilité, essai d'installation d'une version antérieure (pb avec version de Python?)

In [6]:
!pip install spell-checker==1.5.0

     |████████████████████████████████| 2.2 MB 7.8 MB/s eta 0:00:01
  Using cached data_structures3x-1.4.6-py3-none-any.whl (6.3 kB)
ERROR: Could not find a version that satisfies the requirement python-algorithms-3x (from spell-checker==1.5.0) (from versions: none)
ERROR: No matching distribution found for python-algorithms-3x (from spell-checker==1.5.0)


In [9]:
import sys
print(sys.version)

3.8.10 (default, Nov 22 2023, 10:22:35) 
[GCC 9.4.0]


Python-alogorithms-3x seulement compatible avec Python 3 et Python 3.9? essayer d'exécuter cette tâche dans un nouvel environnement, avec une autre version de Python? 



Essai correction via ChatGPT, on the same text that was used with Tesseract (The gazette. [volume]_17981210.txt)

Following prompt: "correct the following text, make it legible, keeping in mind that it is an article that was published in the united states in 1798 "* w
4 ' , 4
4 ' 1 r \ » * • v • * , .x - ' ■(. "f \ ' ‘:im f' V t . f- 1
t X
I
_ ,,,, ♦
Cwm“330 By Elezer Alley Jenks, Portland, Maine. t*>wMs i.j
Monday,December io, 1798.
ton THE GAZETTE.
MR. JINKS,
That there fhould exifl in this free
and enlightened countrj the fentiments imputed
to our Jacobins, fo extremely hoftile to all re
ligion, and to every regular conflicted Govern
ment, and fo daringly bent on the propagation
of revolutionary principle*, has exritcd the
wonder, and purged the underilandings of our
frofound.il politicians. The publication of
Rtbifont Proofs of a Confhiracy, in my opinion,
removes every doubt. This author, deferving
of the pitronage of every Amencan, has tr iced
this palCon for innovation to its true fource,
and folemnly warned us of what unthinking
mortals very little expert. I will endeavor, for
the benefit of your readers, to trace in thefe
proofs the prefent diforders of the world.
The nations of Europe have been infefled for
about a century p.»ft with a fet of philolbphers
and reformers, who, under the mafic of liberty
and liberality of fentiment, have endeavored to
fubvert all government, and to annihilate all
religion ; and Voltaire, Helvetia, and Condorcet,
are of this description. Whoever reads their
works, tho clothed in all the charms of wit and
fancy, mud be convinced of their tendency to
this one point. Ti e iatal Jlamina of revolution,
which has filled the w >rld with blood and
tears, wa* generated by thefe men ; and
fomc of them lived to wimefs the fuccefs
of their lihors. They conftamly held up
to view a kind of U.opian felicity, dill ant,
yet fure, whenever mankind could exempt]
Ihemfelves from the fetters of religion, and the
impofitions of ie~al reflraints ; and they adopted
and regulated their condufl by this maxim,
that the end projected would fanrtify the means
Becfffary to accomplifh it_We wiil now ex
plain this end and thefe means, and alio hint at
their origin.
/idam vv sisHAUfT, a profciior in the Uni- j
wefiity at Ingollladt in Bavaria, about the year
1765, invented a plan, calculated, as he con
ceived, to dellroy by a llovv and fubtle operation
all the religions and governments in the world.
His object was to free man from all reftraints
whatever, except that of his reafon, and to re
duce all nations to their primitive (implicity.
For this purpofe he gave his pernicious do»flrine
the name of Maf*nry—-which was calculated to
avoid fufpidom, and to facilitate his deftgns.
He therefore made his court to real tnafons—
informed them that he was able to confer on
them a great number of degrees (which by the
bye were of his own invention) not as yet known
among them. To each of thefe deftgns he at
tached daz/l'ng fymbols and literals, calculated
to captivate the fancy, and to crcafe a defire for
further knowledge in the art. Several of the firft
degrees were intended only to inrtill into the
miud a peculiar m »de of thinking, favorable to
h;s plan ; and by the time the candidate was
introduced into the Illuminati degree, he was
fur editor every evil work. Here the great
dedgn of Weishavpt was fully unfold-d—and
from a regular progrefs in the corruption of the
hea»t, his pupils f<w no caufe to repent of their
illumination. Thefe lodges were formed and
eftabhfhcd over all Germany and France before
the mrlijnancy of their principles we e difeover
ed. The grelt founder of the order di J not ftop
here. He procured men of hi* cwn principles
to b; place i over xnoit of the fem nar es of
learning, fo as to conupt the morals of the
fifing generation. He promoted the eiUblifit
ment °t female lodges, in which the lame fenri
ments were inculcated, well knowing that the
folterje* h ive an almoft invincible control over
the condu& of men. Printers and bookJeller*
were feized with thzjlamiaa> and had a rem ok
able influence. Thus this inhuman projedlorof
national diftrefles, deceived the true mafons—
t and by this very deception eventually inflated
all Europe with revolutionary principles, artd
; with an utter deteftation of religion and morali
ty—and transformed them into brutes and
^ atheifts S
Previoufly to the commencement of the
French revolution, the lodges in France had
i received all the new Tangled degrees of Weisn
j Avr r, ercept that of thc Illuminati. The dawn
of political troubles in th-tt country, induced
them to fend a folemn deputation to theGerman
lodges, and to requcfl a conTeration of this ex
■■ alted degree in mafonry. Deputies were fent
I from Getmany fo Paris to inaugurate their
3 brethren into the fecrct my fteries of the llluwimti
i _which wholly confided in political and re
l gious d’fputes and in plans to deftroy the
^ monarchy of France, and in the place of order
r and C'vil rule to fubiKtute a legal fyftem of
a.heifm, plunder and rapine. The more fully
to eff-cl thefe purpofes, each lodge in Fiance
dcpHted a commitree to meet in the Hill of the
facobin friars, to difeufs poftical and religious
I fubjeifts—to give a tone to the nation, and to
) I influence the national convention, which had
I then began to a flu me the tharafler of a legifla
■' tip* and permanent body. H<.re then we have
»■ the origin r.f French Jacobins, and the » inci
tes by whi-h they are governed. They ar5
- a I
the pupils of Weishaupt, and the profefled
advocates of Atheifm, of robbery, plunder and
murder !
One truth is yet to be told, but little known
in this country, which is, that Talleyrand,
the present miniiter of exterior rela’ions, and
the inveterate defpniler of American property
and lame, was the fecond officer in thefe ///«•
initiated lodges, and the principle fpoke in the
wheel of French politics. By his means, joined
to the exertions of Orleans and Mirabeau. ail the
leaders in the national convention were initi
ated into this fociety—which gave a preponde
rance to every meature propofed in the public
councils. Hence m any of their decrees are
clothed in the very language of Weishaupt.
Witnef* the one againft religion, by wh'ch all
kind of homage to the Deity is denounced as a
cheat, and forbidden—and atheifm ellabliflied
as the c.re$d o! Frenchmen.
from the nature of thefe princ'p'es, and the
efh& they have had in accelera’ing the nations
of Europe to the very apex of diforder and
diflrefi, we may easily difeover the origin of
Jacobins, and the oppofition to executive mta
fures in this country. I know that our Jaco
bins difeiaim even the moft diftant relation to
the French fiaternizets, and affed to defpife
them—and I do not pretend to fay, th it there
is any dired communication between them,
except !t be with the leaders of the two nations
-—who by plans formed in fecret and under
cover of night, imperceptibly dired all the
movements in the great fcheme of revolution.
This fell cm* has already awakened the world to
arms ;—England, Fiance, and America, have
felt the tiTeds of it—and Holland, Geneva,
Switzerland, anJ the capital of Italy bave fallen
victims to it. Thefe principles fpread like a
contagious ficknefs—we are fccretly impelled to
plot the deftrudion ot all religion and govern
ment without know ng the caufe ;—and the
Jacobins of this country are the faithful difei
ples of the European Illuminati. I turn with
dii'guft from this horrid pidure of human de
praviry, and pray to God tor the reftoration of
peace.
NUMA.
Barlow s Infamous Letter !
Concluded from fke laft OauittC.
It is dffi:ult for you to conceive to what a
degree their ftnfibility was carried, on ih's fub
jed, at the beginning of the revolution : it was
i clear that 3 fenfibility of fuch force muft he the
foundation of the mod extravagant affedion ;
if properly nourilhrd, it would have begot a
confidence without bounds ; if flighted, or an
fwered with indifference, it muft end ir. jealou
fy, uncontrolcd fiy the rules of juftice, and blind
to the ligh-t of truth. And what was the con
dud of your Prefident ? Thomas Jefferfon was
your ambaffador in France where his fuperjor
talents and republican principles had rcndereJ
him exceedingly dear to all friends of liberty.
It was well known here that his intention was
to retain his place during the revolution They
wifhed it exceedingly, becaufe both he and they
were fenfible that he would be able to render
the mod effential Jiemces to both countries, by
remaining in Paris during a crifis of fuch mo
mentous expeditions : no cne will deny that
the occafion and the place called for the fit ft
diplomatic talents, and the purcfl republican
vi-tue, that the United States could afford.—
jcnenoo went mm ran* on a itiort leave ot
abfence, with a fixed determination to return
as foon as poffible ; but the Prefidcn* ordered
it otherwife ; and the French believed it was
from a t’ilapprobnion of JefFerfon’s attachment
to the caufe of liberty in France. This opin
ion mrjr have been two hrftily formed, but
they were confirm'd in it. by the Prefi cent’s
naming to the fame place G. Morris, who fur
two w aters pad t{ad filled Paris with invedives
againfl every principle of liberty—who was
presiouflv deteHed by all the leaders of th&rev
o’liition—who was known to be the broker, pro
tedor, and cbrrefpondcnt, of the moll obnox
ions emigrants. L is poffible that Waffiington.
in not fuffering JefFerfon to return, might have
aded from other motives than thofe of enmity
to the French revolution, though no other mo
tives appear ; but his naming Morris, was an
infult that admits of no palliative .• it is vain
to fay he was ignorant of the charader that
this name bore in Paris : he was a wide mouth
bawler, and had been for two years the exag
gerating echo of all the abufe in all Burke’s
pamphlets and all the word papers in Lon
don.
This feene was continued here, to the adon
ifhmcnt of all Euiope, for three years. His
bulinefi was to mifiead the Prefident with re
fped to what was going on in France, to infult
the French nation, and as far as poffible to be
tray them ; for it wis univerfal y believed, &
I have no doubt of the fad, that after the An
drian and Engliih ambafiadors retired from
Paris, Morris adei a fecret agent and fpy for
thofe two cabinets. A hady word or adibn
coming from an ambafiador, though malicious
in itfclf, is not always interpreted to be the lan
guage of the gnverpment that feat him j but a
«
feries of oftentations abufe continued for three
years becomes unequivocal. All Europe, lea
gued againft liberty, conlidered America alrea
dy in the coalition ; and France would at that
time have grouped you among her enenres,
had it not been For feveral circumftances whol
ly adventitious or foreign to the condud of
your cabinet.
l. Some Americans in Paris, of chara^er
far more refpe&able than that of Nf orris, endea
voured, and with a momentary lureefs to con
vince the leaders here, that his conduft when
known in America, mull be dilapproved.
4. France was in want of the trade and pro
vifion of the United States, both for her colo.
nies and herf If \ ic would therefore be incon
venient at that time to have them for ene
mies.
3. The con^J u<S of Genet, a fubiefl of fo much
triumph to your cabinet and that of Sr. Jame’s,
was one of the caul'es- that faved you from a
war at that time. Genet had been fent by
Brilfot-— Brifljt was now fallen -- it was for this
reafon the -condu<5t of Genet was difapproved,
and that of the American 'governme t part' d
over in filence, th ugh a filence marked with
refentment and contempt.
When, alter every; rermnitranCe & a f rmal
demtnd from this government, your executive
was pleafed to remove Mo:ris frona his oftenfi
ble fttuation in Ptris ; he emigrated, thtt is, he
went and joined the emigrants in Germany, &
has been ever fince among the enemies ofFrartce.
Hut th's is not all—Ltter from Walh
ingtm to Morris,* dated the latter end of the
year 1795, intercepted and now in the hands of
ihe Direftory, gives him a commitfion as a fe
cret agent to the cabinet of London, to tranfaft
bufmel's fo apparently hoflile to the intereft of
France, that I am allured this letter has (harp
ened the edge of reftntment here more than
the whole of Jay’s treaty. This and other
j ciicumflances have given full credit to the opin
ion here, that a journey which Morris took from
London to Berlin, in the year 1796, was a m<f
fion on the part of the Btitila government to
engage the King of PrufTn to rejoin the coali
tion again ft France Another fa<5h, iho* of left
confequence, could not efcape the animadver
Pi n of the Ticnch government. John Parifh,
American Conf>1 at Hamburg, was employed
by the Englifh government as their agent for
transmitting the ftihfidies and loans to the Em
peror and the King of PrulCa, for the war
agaiaft France j and to freight and fit out v:f
fels for the tranfporting troops to the Weft-In
dies. It may be faid rhat the American gov
ernment were not anfwerable for a thing of this
fort, of which *hey co-ild have bo knowledge f
hut this has not prevente 1 the fafl from being
recognized among the proofs of an unfriendly
difpefition on your fide 1 and certainly great
allowance ought to be made for the j 'alouly of
a nation goaded by all Europe, totmented by
hero-vn traitors, and Handing alone in a canfe
in which (he expeAed, at lead, a friendly coun
tenance from us, if not an a£Lvt fuppoit. She
looked upon the caufe of 1 berty as our caufe ;
and though fhe did not require us to take arms,
the confidcred herfclf as fighting our battles in
her own.
Much has been faid on the fubjeft of national
gratitude, and to afeertain how much, or whe
ther any, was due from us to France, for the
part fiie took in the American war. -
I will not add to the obftrvations that have
been made on this head ; but it is clearly my
opinion, that the' has rendered us mor e fb!»d fer
vice by eftabl thing (he principle of reprefenta
tivc government in Europe, th-n by aiding uj
in America*
I Ihall fry very little on the m'ffion of Mon
roe, becaufe Itake it for granted, from what [
have heard, that he has already told h s own
dory in prirt. I will only fay, that in the
mi Jit of all the difficulties created by the mad*
neft of his predeceflbr, the continued folly of
your Executive, the unfortnna’e conclufton and
ratification of the Eng’ilh treaty, he conduced
himfelfin fuch a manner as to form by his Tin
gle chara&cr a counrerpoife to all the weight
of rcfenrment from this government} nobody
doubts here, but that he would have continued
to do fo to the end of the war, if your cabinet
had let him alone, and confined their blunders
to their own continent. What rand then have
been the sftonifhment of all our freuds, and
the exultation of the court of London, to fee
him recalled in the moll abrupt an 1 cenforious
manner.
For the perfonal qualities of Gen. Pincknev.
becaufe they had little or nothing to do with
his being rtfufed here, as the lucccflor of Mon
roe, I will excufe him for writing weak and
idle letters, but I will nnt excufe your executive
for printing them.f Being rejefted as amb.nf
frdot, he **rcnt to fpend the winter in Holland ;
and all the world knows how many carriage
whee’s it cod him, to make thefe journies thro*
* We have never before heard of any fuch fetter, and
believe n*ne fuch ever raided—it is unqueiliouably one
of the million of Jacobinic lie* invented with deficit to
make our btlovcd Wafliiugton odious to hit country*
men. " t *
t We trull our F*erntive w;i! try to get along with
i out the vat Mr Barlow, cxcufc.
this frightful republican territory : Ncrwith
(landing all thefe evils, both real and imaginar
there ftiil remained one more—to the patience
of this very impatient government, they knew
that Wafliingtor. was in thedotige of his natu
ral life, and netr the clofe of his political career ;
they indulged the hope that when he (hould be
out of office, the American people would come
to their fenfcs, or,- at lead, they faw that the
character of the new Prtltdent Would be a crite
rian by which the decided friend (hip or enm ty
of the United States to France would be clearly
feen. The candidates were Adams'and JefFer
fon j the one a reputed Roy did, -anl enemy
to France *, the othea an emirent Republican,
and a friend to the caufeof 1 be>ty in all coun- .
tries. The ferttimepts rf thefe two men were
known here ; thTe of the people were no: yet
known, becanfe it was fuppofed that the gene
ral idolatry for Wbfh mttpp h id prevented them
from being freely uttereJ \ thefe were the rea
f »ns why the D reft ry determined to tike no
decided 11 cp in co f quence of Monroe** recal,
until the public voice ih u d make between thefe
two candid ites. This accounts for the interf
•which the French pemed to take in the event f that
election. Thett wishing you to e!e3 Jefferfon,
proves that they did not w v.t to quarrel with you,
an l that they fill hoped that the people of America
were friends to liberty. The government here
waited the event. This was an awful paufe in
the American affairs in Europe ; and it is aflton
ilhing to me how you could fail to view it in
that light in America, and to take the mea'ure
which the mr>ft moderate (hare of common fenfe,
and the moll palpable felf inter eft*, poin'ed
out.*
When the eletfSon of Adams was announced
here, i: produced the order of the 2d of March,
which was meant to be little ftiort of a declara
tion of war; but it was fo far flinjt of it as to
leave room on your fide to come forward with
an additional prote^ of negociatioa/jf you wifh
cd to avo'd that Calamity.
The enmity of the old Prefident towards
France, was now confidered as national zed in
America, and the government here was determ
ined to fl.ece you of your property, to a fufH*
cient degree to bring you to your feeling in the
I only nerve in which it was prefumed your fenli
bilt:y lay, which was you pecuniary ir.tereft;
This uncomplying difp "iition of the Direc
tory induced Mr. Adams to call an extraordi
nary meeting of Congrefs, and coufcqu;r.tly to
make a fpeeth.
To a man who had tile lead pretealions to
prudence, there was but two courfes to be tak
en ; one was to declare war, if he wifhed to
ruin his country j the other was, if he w:flied
to fave it, to offer to regociate, by fending fome
man or men that he knew would be agreeable
to France ; or, at lead, not to play-the bully,
by forcing a man back who bad juft been diiven
out of Paris. True po'icy would have been to
retiieve the tniftake of Washington, by fending
back Morroe. You cannot imagine the effeft
produced here by the name only of a known
friend to liberty in America. A report prevail
ed here for a few days, that Madifon was nam
ed to *his million ; it alarm!! dif irmed the gov
ernment of ali refentment. H id the news
proved true, and Madifon arrived, the bufinefs
would have been fettled in 24 hours. But
Adams, to attain l.isobjtdt, whatever it mi ht
be, found cut a third courfe. which difeove s
more invention ihm I fuppofed him to prffefsN
He ioimed a commidion of t ree, to-mak -he
people of she U ited S »tes believe that a ne
gotiation was offered on their part; aud tVn
filled it up wi h names fr m whuh there could
not be the lead expiation of i'ucorf*. The
firil was a man who Isa i juit been rJoted, ;>nrl
could not be offered again without an r iut;
| fending him back was unJcubt cd’y in e red s
: an infult, and it was io received : ’the fee vs d
j was a man whole effigy ha d been burnt in Vir
ginia, for his violent defence ot the Bri iih trea.
j ty, at leaft it was fo reported and believed here ;
i the third was a fi ve ’/take weigh man, appoint*
| ed with the intemi.on that he fhoil'd 1. »v; r.o
influence j and yet, to prove to y< u th* faciiit .’
of this goverrauent, after all that had p if d,
I am able to aflure you, ft cm the heft an horir ,
that if Gerry had been fent alone, an t n t
(hackled with the other two, the Ditefb ry
would have negotiated with h’m wj;hout any
difficulty ; at prefent the ‘three have been here
five months, without being received or rejected %
and a new law is made, by which an additional
number of neutral vdffwh will fail into the hands
of the French/’
I fhcniid hardly gain credit with you were I
I to ftalc on how final] a pivot the fate ot n.st'ons
I turn in Pat is at this moment.
Thefpeech of John Adams, at the opening
; of Congrcfi in November, was waited for he .a
j with as much expectation as if pt.:c? or war
depended upon it. It was hoped that after he
j had fcr.t his eommifiioners, he would at leaft
| avoid the ule ofinfuli'mg language againft the
, nat’on with whom he was pretending to treat.
| But when wc fouud him borrowing thelanguage
* We had the unbroVen fpirlt of JnAp'*4rnt Jmr iunu,
‘ ami dared to aA im conformity «, the “ Jif.’ch.*:*
■ J&T of Fiance noiwithtUadiug.*
i"